stable/concourse/templates/worker-statefulset.yaml

```diff
@@ -163,3 +163,6 @@ spec:
   updateStrategy:
     type: {{ .Values.worker.updateStrategy }}
 {{- end }}
+{{- if and (eq .Capabilities.KubeVersion.Major "1") (ge .Capabilities.KubeVersion.Minor "7") }}
```

Comment:
We officiall support the current and previous minor versions, which would be 1.7+. The if would not be required.

Reply:
Cheers, Updated.